### PR TITLE
New Top bar

### LINKED ISSUE #89: Top bar
screen shot 2018-05-21 at 11 14 50

But more in the style of: https://webassembly.studio/ (but light, not dark).

### PR DESCRIPTION
**Pull request by ydlamba**
**Top bar Preview:**
screenshot_20180621_042624

**Full editor Preview:**
screenshot_20180621_042936

Fixes #89.

### CODE REVIEW

File: src/components/app.css [hunk 1]
```diff
@@ -1,10 +1,15 @@
+/* Import font style*/
+@import url("https://fonts.googleapis.com/css?family=Roboto");
```
Comment (domoritz) [inline src/components/app.css]: Let's restrict us to latin. Makes the import smaller.

Comment (ydlamba) [reply src/components/app.css]: I think by default it's restricted to latin only.

**Comment by ydlamba** [comment]
**Updates**

- Buttons and IDL logo order changed.

**Sneak Peak**
screenshot_20180621_075242

**Full Editor**
screenshot_20180621_075901

**Comment by domoritz** [comment]
I think the export icon should be the other way around.

screen shot 2018-06-20 at 20 07 33

**Comment by domoritz** [comment]
- [ ] Make the font smaller
- [ ] The run button disappears when auto parsing is on
- [ ] Hover doesn't look great (I think the whole height should be colored

screen shot 2018-06-20 at 22 15 56

**Comment by domoritz** [comment]
* [ ] Support switching between Vega/Vega-Lite
* [ ] Clear should add the right $schema

**Comment by ydlamba** [comment]
screenshot_20180623_014848

File: src/components/app.css (hunk 1 above)
Comment (domoritz) [reply src/components/app.css]:
```
/* cyrillic-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu72xKKTU1Kvnz.woff2) format('woff2');
  unicode-range: U+0460-052F, U+1C80-1C88, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;
}
/* cyrillic */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu5mxKKTU1Kvnz.woff2) format('woff2');
  unicode-range: U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
}
/* greek-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu7mxKKTU1Kvnz.woff2) format('woff2');
  unicode-range: U+1F00-1FFF;
}
/* greek */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu4WxKKTU1Kvnz.woff2) format('woff2');
  unicode-range: U+0370-03FF;
}
/* vietnamese */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu7WxKKTU1Kvnz.woff2) format('woff2');
  unicode-range: U+0102-0103, U+0110-0111, U+1EA0-1EF9, U+20AB;
}
/* latin-ext */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu7GxKKTU1Kvnz.woff2) format('woff2');
  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
}
/* latin */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu4mxKKTU1Kg.woff2) format('woff2');
  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
}
```

We don't need all of these, do we?

**Comment by domoritz** [comment]
- [ ] add some space between the switcher button and clear
- [ ] grey out run button when auto parsing is on
- [ ] should work on smaller screens (tweak css)
- [ ] Reduce roboto to only what we really need
- [ ] Make it work with fullscreen (currently top bar is over the image)

**Comment by ydlamba** [comment]
@domoritz PR is updated! Let me know if I missed anything (hopefully not).

File: src/components/app.css (hunk 1 above)
Comment (ydlamba) [reply src/components/app.css]: Oh! I didn't know that.

**Pull request merged**